src/content/guides/language/sound-problems.md

```diff
@@ -494,6 +494,35 @@ var maximumOrNull =
 
 <hr>
 
+### Conflicting Superinterfaces
+
+A class which `implements` more than one superinterface must be able to
+implement valid overrides for every member.
+Each member with a given name requires compatible signatures across the
+superinterfaces.
+
+Superinterfaces must not include conflicting generics.
+A class cannot implement both `C<A>` and `C<B>`, including indirect
```

Comment:
**nitpick:** You can use `can't` here. In fact, [Google Style prefers it](https://developers.google.com/style/contractions#negation-contractions).